modules/openapi-generator/src/main/resources/python/api_client.mustache

```diff
@@ -160,20 +163,24 @@ class ApiClient(object):
             # use server/host defined in path or operation instead
             url = _host + resource_path
 
+        _decode_utf8 = {k: v != 'file' for k,v in six.iteritems(response_types)}
```

Comment:
`... for k,v in ...` causes build error on Travis CI, it should be:
```suggestion
        _decode_utf8 = {k: v != 'file' for k, v in six.iteritems(response_types)}
```

Reply:
Thanks